plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/TestPrometheusIntegrationStatus.java

```diff
@@ -106,4 +106,14 @@ public void testPushDown()
         MaterializedResult results = computeActual("SELECT * FROM prometheus.default.up WHERE timestamp > (NOW() - INTERVAL '15' SECOND)");
         assertThat(results).hasSizeBetween(1, 2);
     }
+
+    @Test
+    public void testPrometheusWithoutLabels()
+    {
+        // query for Prometheus connector without labels column
+        MaterializedResult results = computeActual("SELECT timestamp,value FROM prometheus.default.up LIMIT 1)");
+        assertEquals(results.getRowCount(), 1);
+        MaterializedRow row = results.getMaterializedRows().get(0);
+        assertEquals(row.getFields().size(), 2);
```

Comment:
It would be better to assert field values in addition to the size.

Reply:
I have updated and pass the tests locally , pelase check it.  Thanks